hashicorp/dev-portal
# chore: remove ai chatbot

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-leah-choreremove-ai-chatbot-hashicorp.vercel.app/) 🔎
- [Asana task](https://app.asana.com/0/1207947026228781/1208048365454088/f) 🎟️

## 🗒️ What

This PR removes the AI chatbot feature on the developer site. It was decided that this feature will no longer be on the site. 

## 📸 Design Screenshots



Before:
<img width="831" alt="Screenshot 2024-08-14 at 11 21 55 AM" src="https://github.com/user-attachments/assets/86daa828-5f96-45bc-b6ff-d13371abfeab">


After:
<img width="838" alt="Screenshot 2024-08-14 at 11 22 08 AM" src="https://github.com/user-attachments/assets/14d233c4-90b4-4cc4-9248-6dbc0bded147">


## 🧪 Testing


1. Go to [preview](https://dev-portal-git-leah-choreremove-ai-chatbot-hashicorp.vercel.app/)
2. command/control k to open menu
3. Verify AI command at top is no longer there
